src/commands/analysis/run-analysis.ts

```diff
@@ -74,7 +74,7 @@ async function runAnalysis(scriptName: string | undefined, options: { environmen
     return process.exit();
   }
 
-  const account = new Account({ token: config.profileToken, region: "usa-1" });
+  const account = new Account({ token: config.profileToken, region: !process.env.TAGOIO_API ? "usa-1" : "env" });
```

Comment:
Run command still has some issues. I tried running "test-analysis.ts" that is attached to an analysis called "test" in TagoIO Deploy. The analysis was correctly placed in external, but the command is trying to use a different analysis in the profile (Bulk Device Create).

<img width="841" alt="image" src="https://github.com/user-attachments/assets/9614778b-efde-406d-91e4-27e53ca351ec">